tox.ini

```diff
@@ -149,3 +149,15 @@ deps =
   {[testenv:make-changelog]deps}
 isolated_build = {[testenv:make-changelog]isolated_build}
 skip_install = {[testenv:make-changelog]skip_install}
+
+[testenv:deps]
+description = Bump all test dependencies
+skip_install = true
+basepython = python3.10
+deps =
+  pip-tools>=6.5.1
+setenv =
+  # without his upgrade would likely not do anything
+  PIP_CONSTRAINT = /dev/null
+commands =
+  pip-compile -r test-requirements.txt -r docs/requirements.in -o requirements.txt
```

Comment:
Do not make docs builds depend on unrelated deps. The lockfile must be separate.